README.md

```diff
@@ -236,9 +236,43 @@ Build client in its directory (`cd client`):
     npm install
     npm run build
 
+### Windows build
+
+Clone the repository with:
+
+    git clone https://github.com/izderadicka/audioserve
+
+or download the ZIP archive. 
+
+Download the `x86_64-pc-windows-msvc` version of Rust compiler from [rust-lang.org](https://forge.rust-lang.org/infra/other-installation-methods.html#standalone). Also download and perform the default installation of the [Build tools for Visual Studio](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16). You may ignore the request to reboot the system after the installation.
+
+Download the latest "development" version of Windows 64-bit binaries of FFmpeg from [the official website](https://ffmpeg.zeranoe.com/builds/win64/dev/). Extract all `/lib/*.lib` files such as `avcodec.lib` from the archive to the `C:\Program Files\Rust stable MSVC 1.43\lib\rustlib\x86_64-pc-windows-msvc\lib` folder.
```

Comment:
I'm not sure if all latest versions will work as binding is generated for ffmpeg 4.1   - so I'd rather recommend to use 4.1 -  for more recent versions we might see problems like segfault.

Reply:
fixed that.